src/main/java/com/conveyal/gtfs/storage/ObjectRelationalMapping.java

```diff
@@ -0,0 +1,163 @@
+package com.conveyal.gtfs.storage;
+
+import com.conveyal.gtfs.model.Entity;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import java.net.URL;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.Statement;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Created by abyrd on 2017-03-27
```

Comment:
We should not reinvent the wheel by creating an ORM.  👎

Reply:
This was an early experiment and has been removed.